conftest.py

```diff
@@ -416,6 +416,20 @@ def _benchmark_all(client):
 # ############################################### #
 
 
+@pytest.fixture(scope="session")
+def gitlab_cluster_tags():
+    tag_names = [
+        "GITHUB_JOB",
+        "GITHUB_REF",
+        "GITHUB_RUN_ATTEMPT",
+        "GITHUB_RUN_ID",
+        "GITHUB_RUN_NUMBER",
+        "GITHUB_SHA",
+    ]
+
+    return {tag: os.environ.get(tag, "") for tag in tag_names}
```

Comment:
I asked offline, but asking here for completeness: is there a config option we can set for specifying tags globally, instead of needing to pass these through to each cluster instance? 

The answer was no -- though it would be useful in this particular case

Reply:
No. It's a feature-enhancement we could add without too much trouble, obviously if we added we could later change this code to use that.